WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -288,7 +288,7 @@ private void setupTabs() {
             mTabLayout.addTab(mTabLayout.newTab().setText(R.string.media_videos)); // FILTER_VIDEOS
             mTabLayout.addTab(mTabLayout.newTab().setText(R.string.media_audio)); // FILTER_AUDIO
 
-            mTabLayout.setOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
+            mTabLayout.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
```

Comment:
Since we only want a single listener in this case, we may want to clear the observers of the `mTabLayout` before adding this listener by doing `mTabLayout.clearOnTabSelectedListeners();` before this.

I don't think this would make a difference right now, but if for some reason this code is called more than once during the lifecycle of the activity, we'd have more than one listeners.

It's not strictly necessary, so it's your call whether to add it or not.

_P.S: Unrelated to this PR, we probably want to call `mTabLayout.removeAllTabs();` before `addTab` calls as well for the same reason._

Reply:
It shouldn't get called twice, but you're right. Defensive programming is definitely a good idea in a project so big.

@oguzkocer Should I add that call for the tabs? Or do we want to create a new issue for it?

Reply:
We can do it in this PR if you wouldn't mind.